frontend/cookie.ts

```
@@ -0,0 +1,61 @@
+export function setCookie(name: string, value: string) {
+  return Effect.gen(function* (_) {
+    const encodedName = yield* _(encodeUriComponent(name));
+    const encodedValue = yield* _(encodeUriComponent(value));
+
+    document.cookie = `${encodedName}=${encodedValue}; path=/;`;
+  });
+}
+
+export function getCookie(name: string) {
+  return Effect.gen(function* (_) {
+    let encodedName = yield* _(
```

ESLint

[error] 'encodedName' is never reassigned. Use 'const' instead.

(prefer-const)